projects/addon-commerce/utils/get-currency-by-currency-code.ts

```diff
@@ -0,0 +1,55 @@
+import {TuiCurrency, TuiCurrencyCode} from '@taiga-ui/addon-commerce/enums';
+
+// eslint-disable-next-line complexity
+export function tuiGetCurrencyByCurrencyCode(currency: TuiCurrencyCode): TuiCurrency | null {
+    switch (currency) {
+        case TuiCurrencyCode.Ruble:
+            return TuiCurrency.Ruble;
```

Comment:
Maybe object ?

```ts
export const VARIABLE_NAME = {
    [TuiCurrencyCode.Ruble]: TuiCurrency.Ruble,
    [TuiCurrencyCode.Dollar]: TuiCurrency.Dollar,
    // ...
};
```

Reply:
Right, sounds simpler. We used switch case in getting currency symbol because it worked with both code and currency, so 2 cases led to the same result.